source/_drafts/Feature-Spotlight-Accessibility.md

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Feature Spotlight: Accessibility'
+author:
+  name: Brandon Casey
+  github: brandonocasey
+tags:
+  - video.js 6.0
+  - accessibility
+  - a11y
+---
+
+Accessibility! The most important feature you never knew about.
+
+In the Video.js organization we try hard to have good accessiblity. Like most other software any change can affect the system in unintended ways. For example `MuteToggle` and `VolumeControl` were recently (in Video.js 5) married into `VolumeMenuButton`. While this change did allow these controls to work in tandem, it also did something unintended. It broke accessibility. In this post we will go over what broke, what the fix was, what accessibility is, and how to test and make sure it works.
+
+> Feel free to skip to the [last section](#The-problem-and-the-solution) if you already know what accessibility is.
+
+### Accessibility? What's that?
+Accessible software has support for users with vision, hearing, or other impairments. It also helps users that just want to use the keyboard to navigate. Out of the box web applications have some accessibility due to the nature of HTML, but this is only the case if you are using native elements in intended ways. If you cannot use native DOM elements, like button, and instead must use a div for buttons, then you need worry about accessibility in your page.
+
+Supporting hearing impairment is not something that we can do for the users of Video.js. Instead we must indirectly support these users by supporting captions in their videos. In Video.js we have had support for captions and subtitles for some time, internally they are called `TextTracks`.
```

Comment:
Might be worth mentioning the transition to supporting WebVTT format text tracks in v4.??, from supporting SRT format?